lib/mi_bridges/driver/relationship_information_page.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class RelationshipInformationPage < BasePage
+      TITLE = "Relationship Information"
+
+      def setup
+        @first_member = find_first_member
+        @second_members = find_second_members
+      end
+
+      def fill_in_required_fields
+        second_members.each do |member|
+          fill_in_relationship(member)
+          buy_food_with(member)
```

Comment:
Since you're passing a `member` to all of these private methods, I wonder if these two procedures might want to be their own object, like `SubmitRelationship.new(member).run` and `BuyFoodWith.new(member).who` or something?

Not a huge deal, just passing the same type of object to lots of private methods always smells like "new object" to me.

Reply:
yeah I totally agree with that, and I think we should do it! I am going to merge this as-is and then do a follow up PR